ext/rbkit_server.c

```diff
@@ -347,6 +347,14 @@ static VALUE stop_stat_server() {
 }
 
 static VALUE send_hash_as_event(int argc, VALUE *argv, VALUE self) {
+  /*
+   * Check if the server connection is available before trying to pack GC stat
+   * information into msgpack buffers. By checking if the logger variable is
+   * initiated or not, we achieve this. If the server has been stopped, we free
+   * the logger pointer and set it to 0. Refer to `stop_stat_server` method.
```

Comment:
I feel this comment can be made a bit more brief. Maybe something like "No-op if server is not running, because this function is exposed as a method on the Rbkit class and can be called by the user".